charts/vela-core/templates/defwithtemplate/webservice.yaml

```diff
@@ -185,6 +185,10 @@ spec:
         						command: parameter.cmd
         					}
 
+        					if parameter["args"] != _|_ {
```

Comment:
I think you should not edit the two Definition yaml files. 
Because here is comment in this file "# Code generated by KubeVela templates. DO NOT EDIT. Please edit the original cue file.
"

Reply:
it's automatically generated by `make reviewable`

Reply:
we should check in the automatically generated result